feat(2092): [2] Dropdown build parameters (#398)
screwdriver-cd/screwdriver-cd.github.io: docs/user-guide/configuration/parameters.md

      active: true
---
## Defining Parameters

> There are 3 ways to define parameters, you can see them both in the example below.

Reviewer: ```suggestion
There are 3 ways to define parameters, you can see them in the example below:
```

```yaml
parameters:
    nameA: "value1"
    nameB:
        value: "value2"
        description: "description of nameB"

>     nameC: ["value1", "value2"]

Reviewer: should we add that the default will be the first value in the array?

Author: Yes, and It is important thing. I add about that.

Author: Hi @tkyi. Is it OK?

Reviewer: small comment!

These two examples above are equivalent.

> You can also define parameters with array to pick up from some candidates. The top value of the array is picked up as default, e.g `value1` is selected as default in the example blow:

Reviewer: ```suggestion
You can also define parameters in an array for drop down options. The first value in the array is picked up as default, e.g `value1` is selected as default in the example below:
```

Author: Thank you. I fixed it.
